### PR TITLE
ch01: add portable mode

### PR DESCRIPTION
**Pull request by rating89us**

### CODE REVIEW

File: en/ch01.md [hunk 1]
```diff
@@ -23,12 +24,15 @@ You'll see a list of available languages, click on the one of your choice to cha
...
 (3) **Advanced mode:** In this mode your wallet will display all advanced features available, like changing fees/transaction priority, solo mining, message verification, transaction proof/check, offline transaction signing, key images importing, custom remote node (including .onion remote nodes), custom bootstrap node, starting a local node without setting a bootstrap node (maximum privacy), changing blockchain location, and daemon log. In this mode you can choose if you want to connect to the Monero network via a remote node or a local node. If you decide to run a local node, the blockchain file will be downloaded. If you want to start using Monero immediately, you can also set a bootstrap node (remote node) in the Settings > Node page. To achieve maximum privacy in your connection to the Monero network, it's recommended to avoid using a remote/bootstrap node and instead use advanced mode to start a local node without setting a bootstrap node.  
-(4) **Change language:** This will let you choose the language from the startup page (again).
+
+Optional features
```
Comment (erciccione) [inline en/ch01.md:33]: Maybe better to make this a header?

Comment (rating89us) [reply en/ch01.md]: added a subheader style

**Comment by erciccione** [comment]
conflicts

File: en/ch01.md [hunk 2]
```diff
@@ -23,12 +24,16 @@ You'll see a list of available languages, click on the one of your choice to cha
...
-(4) **Change language:** This will let you choose the language from the startup page (again).
+
+**Optional features:**  
+
+(4) **Portable mode:** choose this mode if your Monero GUI executable is on a USB stick, an external disk drive or any other portable storage medium. When enabled, this mode will store your settings and create new wallets in the same folder of your Monero GUI executable, which means Monero GUI will not leave any traces (files) on the computer(s) you use to open your wallet files, and you will not have to reconfigure Monero GUI every time you open your wallet file in a different computer. But beware: selecting this mode will not move existing wallets into your Monero GUI executable folder, and it will not erase traces that may already have been left on your operating system  from previous installs (for instance, if you have previously used Windows installer to install Monero GUI in Program Files folder).
+=======
```
Comment (erciccione) [inline en/ch01.md:36]: leftover from the rebase.

**Review by erciccione — APPROVED** [review]

**Pull request merged**